crates/l3/appchain-core-contract-client/tests/test_client.rs

```diff
@@ -0,0 +1,37 @@
+use std::sync::Arc;
+use color_eyre::Result;
+use starknet_instance::StarknetClient;
+use starknet_ff::FieldElement;
+use appchain_core_contract_client::clients;
+use clients::client::StarknetCoreContractClient;
+use appchain_core_contract_client::interfaces::operator::Operator;
+
+#[tokio::test]
+async fn test_operator() -> Result<()> {
+    // Create a Starknet client
+    let rpc_endpoint = "http://localhost:5050".to_string();
+    let chain_id = "0x4b4154414e41".to_string();
+    let priv_key = "0x1800000000300000180000000000030000000000003006001800006600".to_string();
+    let account_addr = "0x6162896d1d7ab204c7ccac6dd5f8e9e7c25ecd5ae4fcb4ad32e57786bb46e03".to_string();
+    let starknet_client = StarknetClient::attach(rpc_endpoint, chain_id, priv_key, account_addr)?;
+
+    // Create a StarknetCoreContractClient
+    let address = FieldElement::from_hex_be("0x07100a5f88487652c9efaa81d9037927f67000512ff25e00dded29e5af58ae58")?;
```

Comment:
is this supposed to be predeployed on devnet? if yes, I think it's better to deploy it as a part of the test no? otherwise how do we test it in CI

Reply:
I will create another issue in which i will write tests in a sandboxed devnet environment.